lib/ace/test/mockdom.js

```diff
@@ -626,7 +626,14 @@ function TextNode(value) {
     });
 }).call(TextNode.prototype);
 
-var window = {};
+var window = {
+    get innerHeight() {
+        return WINDOW_HEIGHT;
+    },
+    get innerWidth() {
+        return WINDOW_WIDTH;
+    }
```

Comment:
Why do we need getters here, not just
```
var window = {
    innerHeight: WINDOW_HEIGHT,
    innerWidth: WINDOW_WIDTH
}
```
?

Reply:
To prevent tests from modifying these properties in a way that would not work when running the same test in a real browser.